homeassistant/components/cert_expiry/sensor.py

```diff
@@ -77,7 +77,6 @@ async def async_setup_entry(
 class CertExpiryEntity(CoordinatorEntity[CertExpiryDataUpdateCoordinator]):
     """Defines a base Cert Expiry entity."""
 
```

Comment:
Isn't this missing the `_attr_translation_key`?

Reply:
This super class is never used, only the child classes

Reply:
Ok, thx, now I saw that 👍